sdk/react/utils.js

```diff
@@ -129,84 +129,84 @@ var Utils = {
   stringForErrorCode: function(errorCode: int) {
     switch (errorCode) {
       /* Authorization failed - TODO add to language files */
-      case 0: 
+      case 0:
         return "Authorization failed";
       /* Authorization response invalid - TODO add to language files */
       case 1:
         return "Invalid Authorization Response";
-       /* Authorization heartbeat failed */  
+       /* Authorization heartbeat failed */
       case 2:
-        return "INVALID HEARTBEAT"
-       /* Content tree response invalid - TODO add to language files */  
+        return "Invalid Heartbeat"
+       /* Content tree response invalid - TODO add to language files */
       case 3:
         return "Content Tree Response Invalid";
-       /* Authorization signature invalid - TODO add to language files */  
+       /* Authorization signature invalid - TODO add to language files */
       case 4 :
         return "The signature of the Authorization Response is invalid";
-      /* Content tree next failed - TODO add to language files */    
+      /* Content tree next failed - TODO add to language files */
       case 5:
         return "Content Tree Next failed";
-      /* Playback failed */    
+      /* Playback failed */
       case 6:
-        return "PLAYBACK ERROR";
-      /* The asset is not encoded */    
+        return "Playback Error";
+      /* The asset is not encoded */
       case 7:
         return "This video isn't encoded for your device";
-      /* Internal error - TODO add to language files */    
+      /* Internal error - TODO add to language files */
       case 8:
         return "An internal error occurred";
-      /* Metadata response invalid */    
+      /* Metadata response invalid */
       case 9:
         return "Invalid Metadata";
-      /* Invalid authorization token */    
+      /* Invalid authorization token */
       case 10:
-        return "INVALID PLAYER TOKEN";
-      /* Device limit has been reached */    
+        return "Invalid Player Token";
+      /* Device limit has been reached */
       case 11:
-        return "Device limit has been reached";
-      /* Devuce binding failed */    
+        return "Authorization Error";
```

Comment:
Do we not need the device limit error message anymore? Did it move to a different number? I just don't see it.

Reply:
I removed it as required by Product (Steve, Nikhil) They want the message to be Authorization Error. We still send device limit error to the message bus is just different how we show the message in the SkinSDK.